app/server/src/tags/tags.txt

```diff
@@ -0,0 +1,740 @@
+Application
+Comprehension
+Interpretation
+Social science
+Humanities
+Religion
+Classics
+Cybersecurity
+Robotics
+Astrophysics
+Cosmology
+Geophysics
+Immunology
+Virology
+Conservation
+Wildlife biology
+Forestry
+Marine biology
+Speech therapy
+Sports science
+Kinesiology
+Pharmacy
+Recreation
+Tourism
+Hospitality
+Culinary arts
+Film and video production
+Broadcasting
+Costume design
+Stage management
+Algorithms
+Data structures
+Computation
+Programming languages
+Software development
+Database systems
+Computer architecture
+Operating systems
+Computer networks
+Data communication
+Cloud computing
+Internet of Things
+Mobile computing
+Distributed systems
+Parallel computing
+Grid computing
+Cluster computing
+High-performance computing
+Machine learning
+Deep learning
+Neural networks
+Natural language processing
+Computer vision
+Data mining
+Information retrieval
+Big data
+Data engineering
+Network security
+Information security
+Application security
+Cloud security
+Internet security
+Database security
+System security
+Mobile security
+IoT security
+Computer forensics
+Ethical hacking
+Penetration testing
+Vulnerability assessment
+Risk management
+Compliance
+Governance
+Data privacy
+Web development
+Business skills
+Project management
+Creative skills
+Educational technology
+Health and wellness
+Personal development
+Language learning
+Data visualization
+Computer graphics
+Game development
+Virtual reality
+Augmented reality
+3D printing
+Blockchain
+Cryptocurrency
+Legal issues
+Social media
+Digital marketing
+Branding
+Customer service
+Negotiation
+Leadership
+Team building
+Communication skills
+Time management
+Goal setting
+Productivity
+Stress management
+Innovation
+Decision making
+Baking
+DIY home improvement
+Weaving
+Ceramics
+Magic
+Card tricks
+Coin tricks
+Escape artistry
+Lock picking
+Sleight of hand
+Collecting
+Antiques
+Comics
+Stamps
+Coins
+Trading cards
+Sports memorabilia
+Models
+Legos
+Puzzles
+Crossword puzzles
+Sudoku
+Word search
+Cryptograms
+Scrabble
+Othello
+Reversi
+Chinese checkers
+Poker
+Blackjack
+Slots
+Roulette
+Craps
+Baccarat
+Bingo
+Lotto
+Keno
+Rummy
+Spades
+Backpacking
+Mountain biking
+Road cycling
+BMX
+Rollerblading
+Wakeboarding
+Kitesurfing
+Stand-up paddleboarding
+Yachting
+Jet skiing
+Water skiing
+Snow skiing
+Alpine skiing
+Table tennis
+Fencing
+Judo
+Karate
+Taekwondo
+Kung fu
+Muay Thai
+Kickboxing
+Boxing
+Brazilian jiu-jitsu
+Mixed martial arts
+Wrestling
+CrossFit
+Zumba
+Aerobics
+Dance fitness
+Boot camp
+HIIT
+Cardio
+Strength training
+Stretching
+Mediation
+Mindfulness
+Qi gong
+Tai chi
+Aikido
+Elementary education
+Middle school education
+High school education
+Adult educations
+Language arts
+Spelling
+Phonics
+Handwriting
+Geometry
+Algebra
+Trigonometry
+Calculus
+Probability
+Civics
+Government
+Ethics
+World languages
+French
+Spanish
+German
+Italian
+Chinese
+Japanese
+Korean
+Arabic
+Russian
+Portuguese
+Latin
+Greek
+English as a second language
```

Comment:
This exists but no Turkish, add it yourself please 🙏

Reply:
Done